037-WindowsVirtualDesktop/README.md

```diff
@@ -0,0 +1,55 @@
+# What The Hack - Windows Virtual Desktop
+
+## Introduction
+
+This hack is designed to help you ramp up for the WVD 140 certification or just to learn Windows Virtual Desktop. With people starting to work remotely WVD has quickly gained adoption across the globe.
+
+The hack starts off by covering the basics of WVD and then digs deep into the different components of WVD and different tyes of solutions you could encounter in a WVD environment.
+
+## Learning Objectives
+
+In this hack you will learn how to set up a Windows Virtual Desktop environment from start to finish. You will take a customer scenario and build it out in your own environment. Once your WVD environment is built you will learn how to scale. monitor and manage the environment with Azure resources.
+
+## Challenges
+
+- Challenge 0: **[Windows Virtual Desktop  Pre-Reqs](Student/00-Pre-Reqs.md)**
+- Challenge 1: **[Planning your WVD Architecture](Student/01-Plan-WVD-Architecture.md)**
+- Challenge 2: **[Implementing and Managing your WVD Network](Student/02-Implement-Manage-Network.md)**
+- Challenge 3: **[Implement and Manage your storage for FsLogix](Student/03-Implement-Manage-Storage.md)**
+- Challenge 4: **[Create and Manage your Images for your Session Hosts](Student/04-Create-Manage-Images.md)**
+- Challenge 5: **[Create and Configure your WVD HostPools](Student/05-Create-Configure-HostPools.md)**
+- Challenge 6: **[Implement and Manage your FsLogix](Student/06-Implement-Manage-FsLogix.md)**
+- Challenge 7: **[Install and Configure your applications](Student/07-Install-Configure-Apps.md)**
+- Challenge 8: **[Plan and Implement BCDR for your WVD Environment](Student/08-Plan-Implement-BCDR.md)**
+- Challenge 9: **[Automation of your WVD Tasks](Student/09-Automate-WVD-Tasks.md)**
+- Challenge 10: **[Monitor and Manage the performance health of your environment](Student/10-Monitor-Manage-Performance-Health.md)**
+- Challenge 11: **[Configure settings for user experience](Student/11-Configure-User-Experience-Settings.md)**
```

Comment:
All of the link texts for this table of contents should match the title header in the challenge text itself. Right now there are some mismatches. Up to you which of the two wins, but they should match.

Also I would capitalize every word in a section header/title except the smallest words, 'a', 'the', 'of'